algos/src/sample_rate_conversion.rst

```diff
@@ -329,18 +324,17 @@ conversion with small edit into src_test.m.
 		+t.plot_close_windows = 0;  % Workaround for visible windows if Octave hangs
 		+t.plot_visible = 'on';     % Use off for batch tests and on for interactive
 		+t.files_delete = 0;        % Set to 0 to inspect the audio data files
-		
+
 		%% Init for test loop
 		n_test = 7; % We have next seven test cases for SRC
 
 
 Tips for debugging
 ==================
 
-As additional information for debugging there's some prints from
-script run to see the command line arguments that src_test.m uses for
-the shell script src_run.sh and the testbench executable. In this case
-the output is:
+Additional debugging information includes prints from scripts run to see the
```

Comment:
Not sure this is any better than mine.. When running the the script "src_test(32, 32, 32000, 48000);" it prints quite a lot and when scrolling up to start the attached code block can be seen. The information is useful because it shows the quite tricky command line arguments for the testbench executable. Also due to complicated testbench build structure the libraries location can be seen from there as well as the test topology that's used. Is this of any help?